cloudwatch-logs-to-logstash/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [ -z $TEST_FUNC_NAME ]; then
+    echo 'TEST_FUNC_NAME is unset';
+    exit 1
+fi
+
+if [ -z $PROD_FUNC_NAME ]; then
+    echo 'PROD_FUNC_NAME is unset';
+    exit 1
+fi
+
+cd lambda
+npm install
+
+npm test
+
+# TODO get or create config.json
```

Comment:
I can't think of a decent way to do this; writing CI env vars to disk doesn't feel that nice.

I'm tempted to leave CI/CD for now...